src/python/pants/backend/codegen/tasks/protobuf_gen.py

```diff
@@ -124,7 +124,8 @@ def execute_codegen(self, targets):
     sources_by_base = self._calculate_sources(targets)
     sources = OrderedSet(itertools.chain.from_iterable(sources_by_base.values()))
 
-    self.assert_sources_present(sources, targets)
+    if not self.assert_sources_present(sources, targets):
```

Comment:
An assertion isn't meant to return a value.   Maybe you want to rename this method. s/assert/validate/ ?